frontend/src/pages/homepage.jsx

```diff
@@ -1,50 +1,39 @@
-import React from "react";
-
-const Homepage = () => {
-    return (
-        <div className="bg-[#0f172a] min-h-screen flex flex-col items-center justify-start py-12 px-6 text-white">
-            {/* Header */}
-            <div className="max-w-lg text-center mb-8">
-                <h1 className="text-4xl font-bold">
-                    THE NEXT GENERATION OF LEARNING
-                </h1>
-            </div>
-
-            {/* Registration and login */}
-            <div className="form-container">
-                <h2 className="form-title">Exam Crack</h2>
-                <form>
-                    <div className="form-group">
-                        <label htmlFor="email" className="form-label">Email*</label>
-                        <input
-                            type="email"
-                            id="email"
-                            className="form-input"
-                            placeholder="Enter your email"
-                            required
-                        />
-                    </div>
-                    <div className="form-group">
-                        <label htmlFor="password" className="form-label">Password*</label>
-                        <input
-                            type="password"
-                            id="password"
-                            className="form-input"
-                            placeholder="Enter your password"
-                            required
-                        />
-                    </div>
-                    <button type="submit" className="form-button">
-                        Log in
-                    </button>
-                    <p className="form-separator">or</p>
-                    <button type="button" className="form-button">
-                        Sign up
-                    </button>
-                </form>
-            </div>
-        </div>
-    );
-};
-
-export default Homepage;
+import React from 'react';
+
+const Homepage = () => {
+  return (
+    <div style={{
+      backgroundColor: '#0f172a', 
+      color: '#ffffff', 
+      minHeight: '100vh',
+      display: 'flex',
+      flexDirection: 'column', 
+      fontFamily: 'Inter, sans-serif'
+    }}>
+      {/* Main Content */}
+      <div style={{
+        flex: 1,
+        padding: '3rem 4rem',
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center',
+        textAlign: 'center',
+      }}>
+        <h1 style={{
+          fontSize: '2.5rem',
+          fontWeight: 'bold',
+          marginBottom: '2rem',
+        }}>THE NEXT GENERATION OF LEARNING</h1>
+        <p style={{
+          fontSize: '1.25rem',
+          color: '#cbd5e1',
+        }}>

```

Comment:
I'd suggest to change this styling to Tailwind classes for better readability (can just ChatGPT that).